pkg/parcacol/querier.go

```diff
@@ -393,7 +393,7 @@ func (q *Querier) queryRangeDelta(
 		logicalplan.Col(profile.ColumnDuration),
 	}
 
-	if isSamplesCount(m.SampleType) {
+	if isSamplesCount(resultType) {
```

Comment:
I would still use `m.SampleType`. It's no different, and using `resultType` only happens to be the same at this point in time, as `m.SampleType` is the "default".